specification.md

```diff
@@ -1794,11 +1875,13 @@ the loop are known, to submit all the jobs at once:
 
 <img width="300pt" src="diagrams/bulk_submission_bulk.svg" alt="Bulk Jobs Timing Diagram"/>
 
-where `send(job_data[])` indicates that we are now sending an array of
+`send(job_data[])` indicates that we are now sending an array of
```

Comment:
Not sure. The "where" is pretty standard to introduce a definition/explanation after a formula/diagram.

Reply:
```suggestion
where `send(job_data[])` indicates that we are now sending an array of
```